### PR TITLE
Add listeners to correctly loads projects in total offline mode

### LINKED ISSUE #139: Projects lists not loading projects if offline (and firebase cache disabled)
If we disable firebase cache and disable internet connection.
The ProjectListActivity won't load the projects from the offline database.
The offline database is correctly loading projects but only a few millisecondes after the first call.

### PR DESCRIPTION
**Pull request by arnauds5**
Add listeners to correctly loads projects in total offline mode.

### CODE REVIEW

**Review by kaourintin — APPROVED** [review]
The bug seems to be fixed on  my machine.

File: app/src/main/java/com/sdp13epfl2021/projmag/adapter/ProjectAdapter.kt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package com.sdp13epfl2021.projmag.adapter
 
-import android.content.Context
```
Comment (PaulCoral) [inline app/src/main/java/com/sdp13epfl2021/projmag/adapter/ProjectAdapter.kt:3]: Won't compile as Context is not imported, and it is used in the constructor

**Review by PaulCoral — CHANGES_REQUESTED** [review]
Won't compile, see comment

**Review by PaulCoral — APPROVED** [review]
Hard to test well on my emulator, but it worked without issue
Fine for me

**Pull request merged**